src/ngx_rewrite_driver_factory.cc

```diff
@@ -151,6 +151,10 @@ RewriteOptions* NgxRewriteDriverFactory::NewRewriteOptions() {
   return options;
 }
 
+RewriteOptions* NgxRewriteDriverFactory::NewRewriteOptionsForQuery() {
+  return new NgxRewriteOptions(thread_system());
```

Comment:
`rewrite_options.h` has: "Most users should explicitly enable the `kCoreFilters` level by calling `SetRewriteLevel(kCoreFilters)`."

I'm not sure why apache doesn't need `SetRewriteLevel`.  I just spent a while looking through the code, and while it looks like `RewriteDriverFactory::InitializeDefaultOptions` is where the default is set for the case I'm testing [1], nginx is also calling `InitializeDefaultOptions` and then seems to use `RewriteDriverFactory::default_options()` in the same way.

[1] Verifying this, if I remove the `SetDefaultRewriteLevel(RewriteOptions::kCoreFilters)` from it then apache starts defaulting to passthrough